src/app/routes/regex/index.test.js

```diff
@@ -175,4 +177,44 @@ describe('mediaRadioAndTvRegexPathsArray', () => {
     ];
     shouldNotMatchInvalidRoutes(invalidRoutes, mediaRadioAndTvRegexPathsArray);
   });
+  describe('mediaAssetPageRegexPath', () => {
+    const validRoutes = [
+      '/pidgin/tori-49450859',
+      '/pidgin/tori-49450859.amp',
+      '/yoruba/media-49450859',
+      '/yoruba/media-49450859.amp',
+      '/punjabi/international-49567825',
+      '/punjabi/international-49567825.amp',
+      '/kyrgyz/sapar-tv-48695523',
+      '/mundo/test_underscore-12345678',
+    ];
+
+    shouldMatchValidRoutes(validRoutes, mediaAssetPageRegexPath);
+    const inValidRoutes = [
+      '/pidgin/tori-494859',
+      '/blah/tori-49450859',
+      '/pidgin/tori-49450859/.amp',
+      '/pidgin/tori-49450859/',
+      '/pidgin/tori-494859.amp',
+    ];
+    shouldNotMatchInvalidRoutes(inValidRoutes, mediaAssetPageRegexPath);
+  });
+
+  describe('mediaAssetPageDataRegexPath', () => {
+    const validRoutes = [
+      '/pidgin/tori-49450859.json',
+      '/kyrgyz/sapar-tv-48695523.json',
+      '/mundo/test_underscore-12345678.json',
+    ];
+
+    shouldMatchValidRoutes(validRoutes, mediaAssetPageDataRegexPath);
+    const inValidRoutes = [
+      '/pidgin/tori-494859.json',
+      '/blah/tori-49450859',
```

Comment:
These should probably all have a `.json` suffix

Reply:
Testing the invalid routes, if the `url` doesn't have `.json` suffix should be an invalid route

Reply:
This example is invalid for multiple reasons, because of `blah` and because it doesn't contain `.json`. This means it could remain false if `blah` mistakenly became a valid match for the `service` part of the route. Ideally we should keep the invalid route examples to a singular purpose